components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -142,8 +161,12 @@ export default function () {
 
             {showPersistentVolumeClaimUI && (
                 <>
-                    <br></br>
-                    <h3 className="mt-12">Workspaces</h3>
+                    <SelectWorkspaceClass
+                        workspaceClass={project.settings?.workspaceClasses?.regular}
+                        enabled={BillingMode.canSetWorkspaceClass(teamBillingMode)}
+                        setWorkspaceClass={setWorkspaceClass}
+                    />
```

Comment:
**thought**: It's a bit confusing that preview environments contain two quite different classes from what's on Gitpod.io and the order is also reversed, but that's out of the scope of these changes, see [relevant discussion](https://gitpod.slack.com/archives/C02F19UUW6S/p1667316174759509) (internal).